lldb/tools/lldb-dap/lldb-dap.cpp

```diff
@@ -411,6 +412,30 @@ void SendStdOutStdErr(DAP &dap, lldb::SBProcess &process) {
     dap.SendOutput(OutputType::Stderr, llvm::StringRef(buffer, count));
 }
 
+static std::string GetStringFromStructuredData(lldb::SBStructuredData &data,
```

Comment:
@JDevlieghere is there a less clunky way to do this? I know we want to handle string creation on the side of lldb-dap so we get some wins not needing to constify, but I don't like my code.